2996_reflection/reflection.md

```diff
@@ -8391,18 +8393,27 @@ To [cpp.predefined]{.sref}:
 
 ::: std
 ```diff
-  __cpp_impl_coroutine 201902L
-  __cpp_impl_destroying_delete 201806L
-  __cpp_impl_three_way_comparison 201907L
-+ __cpp_impl_reflection 2025XXL
+ [...]
+  __cpp_impl_raw_strings            200710L
+  __cpp_impl_ref_qualifiers         200710L
++ __cpp_impl_reflection             2025XXL
+  __cpp_impl_return_type_deduction  201304L
+  __cpp_impl_rvalue_references      200610L
+ [...]
 ```
 :::
 
 and [version.syn]{.sref}:
 
 ::: std
 ```diff
-+ #define __cpp_lib_reflection 2025XXL // also in <meta>
+  [...]
+  #define __cpp_lib_reference_from_temporary  202202L // freestanding, also in <type_traits>
+  #define __cpp_lib_reference_wrapper         202403L // freestanding, also in <functional>
++ #define __cpp_lib_reflection                2025XXL // also in <meta>
+  #define __cpp_lib_remove_cvref              201711L // freestanding, also in <type_traits>
+  #define __cpp_lib_result_of_sfinae          201210L // freestanding, also in <functional>, <type_traits>
+  [...]
```

Comment:
This change is unnecessary, don't need any context.

Reply:
Removed the ones around it.